src/app-layout/visual-refresh/main.scss

```diff
@@ -44,88 +43,6 @@
       }
     }
   }
-
-  /*
-  If disableContentPaddings is not enabled (the majority of use cases) then
-  the following code block will compute the padding behavior for the Main
-  under various circumstances.
-  */
-  &:not(.disable-content-paddings) {
-    @include styles.media-breakpoint-up(styles.$breakpoint-x-small) {
-      /*
-      Notifications is the previous adjacent sibling to the Main in
-      desktop viewports if there are no Breadcrumbs or Header. Use additional
-      to padding on the Main.
-      */
-      &.content-type-default.has-notifications-content:not(.has-breadcrumbs):not(.has-header):not(
```

Comment:
@just-boris I'm going to write a proposal this week for the design team to standardize the `applyDefaults` values across all content types. My impression from @ethangeyer is that this is already the desired goal. If so, we should be able to deprecate this property in Visual Refresh since it will no longer be used for anything.